File: website 16/src/classes/location.ts
import "reflect-metadata";

import { Expose } from "class-transformer";

class Location {
  @Expose() public readonly UUID: string;

  
  @Expose() public readonly Country: string;
  @Expose() public readonly City: string;
  @Expose() public readonly Street: string;

  @Expose() public readonly Latitude: number;
  @Expose() public readonly Longitude: number;

  constructor(
    uuid: string,
    country: string,
    city: string,
    street: string,
    latitude: number,
    longitude: number,
  ) {
    this.UUID = uuid;

    this.Country = country;
    this.City = city;
    this.Street = street;

    this.Latitude = latitude;
    this.Longitude = longitude;
  }
}

export { Location };

File: DataAccess/User/DTOs/location.cs
using DataAccess.Entities;
namespace DataAccess.User.DTOs;

public class LocationDTO
{
    public string UUID { get; set; }

    public string Country { get; set; }
    public string City { get; set; }
    public string Street { get; set; }

    public float Latitude { get; set; }
    public float Longitude { get; set; }


    public LocationDTO(LocationEntity locationEntity)
    {
        this.UUID = locationEntity.UUID;

        this.Country = locationEntity.Country;
        this.City = locationEntity.City;
        this.Street = locationEntity.Street;

        this.Latitude = locationEntity.Latitude;
        this.Longitude = locationEntity.Longitude;
    }
}